### ISSUE TITLE
linux deployed 

### ISSUE THREAD

**Issue by Uangxin** [pynecone-io/pynecone-chat#23]
Can this project be deployed in centos？

**Comment by picklelo** [comment]
Yes, this should work with any OS - are you having issues?

**Comment by Uangxin** [comment]
> Yes, this should work with any OS - are you having issues?

 yes, when I run reflex init, Error :"An error occurred while downloading fnm package: The read operation timed out"

**Comment by adhami3310** [comment]
no reason why this one cannot be deployed with centos or any linux distro, read the reflex self hosting guide
